src/main/webapp/i18n/de/assessment.json

```diff
@@ -90,7 +90,8 @@
                     "showAll": "Alle anzeigen",
                     "showLocked": "Nur gesperrte anzeigen"
                 },
-                "bonusPointExplanation": "In dieser Aufgabe gibt es {{maxPoints}} Punkte und {{maxBonusPoints}} Bonuspunkte, somit können Studierende maximal {{maxPointsWithBonus}} Punkte erreichen, was einem Score von {{maxPercentage}}% entsprechen würde."
+                "bonusPointExplanation": "In dieser Aufgabe gibt es {{maxPoints}} Punkte und {{maxBonusPoints}} Bonuspunkte, somit können Studierende maximal {{maxPointsWithBonus}} Punkte erreichen, was einem Score von {{maxPercentage}}% entsprechen würde.",
```

Comment:
As an example: In dieser Aufgabe gibt es 1 Punkte ... :D (i guess you need to adapt the string according to the point/points)

Reply:
I did not touch this string. Git only marks this line as changed since a comma is added at the end. I agree that this is improvable, but implementing a cumbersome case distinction for this tooltip is not in scope of this PR. I hope you understand this. 😄

Reply:
Yes sure :D